### DISCUSSION TITLE
Welcome to Steps Discussions!

### DISCUSSION THREAD

**Discussion by brittanyarima** [brittanyarima/Steps#68] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

❗️ Just an FYI. I'm held up with getting a couple PR's merged. I apologize for the delay!

**Comment by Drag0ndust** [comment]
Hello, my name is Thimo. I am a software engineer specializing in iOS. In my job, I work on client projects, and in my free time, I develop my own apps. You can check out my [GitHub profile](https://github.com/Drag0ndust) for more details ;)
